### PR TITLE
Add section on IMAGE schema.yaml property

### PR DESCRIPTION
**Pull request by eshiroma**

### CODE REVIEW

File: docs/schema.md [hunk 1]
```diff
@@ -287,6 +287,15 @@ It defines how this object will be handled. Each type has a different set of pro
 
 ---
 
+### type: IMAGE
```
Comment (vcanaa) [inline docs/schema.md]: This only applies for schema v1. Lets add this session here and have a working item to include schema v2 instructions.

Comment (vcanaa) [reply docs/schema.md]: Created #387.

Comment (eshiroma) [reply docs/schema.md]: +1 Thanks for tracking!

**Pull request merged**